src/ripple/consensus/Consensus.h

```diff
@@ -867,15 +867,18 @@ Consensus<Derived, Traits>::timerEntry (NetClock::time_point const& now)
     {
         using namespace std::chrono;
 
+        // checkLCL may change state_, so it needs to run prior to the
+		// switch (state_) below
```

Comment:
Comment indentation is a bit unusual, but if you like it that way I'm good with it.  :+1: